components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/hardware-wallet-connect/types.ts

```diff
@@ -19,6 +20,11 @@ export const HardwareWalletDerivationPathsMapping = {
   [BraveWallet.TREZOR_HARDWARE_VENDOR]: TrezorDerivationPaths
 }
 
+export const SolHardwareWalletDerivationPathLocaleMapping = {
+  [SolDerivationPaths.Default]: 'Default',
+  [SolDerivationPaths.LedgerLive]: 'Ledger Live'
+}
```

Comment:
Can you add a `TODO` or create an issue to localise these strings? Not for this PR, since we already had these hard-coded locale strings.

Reply:
https://github.com/brave/brave-core/pull/15406/commits/70b60595c76911ebb941b0c4c919f393edad01ed

Reply:
Can we also show the path info for both options? Phantom does list the paths.

Reply:
^ Anton and I agreed in DM to create a follow-up issue for considering adding key derivation path details to these options for both Solana and Ethereum.

Reply:
https://github.com/brave/brave-browser/issues/25954

Reply:
Yes that's a good idea. 👍  I set "Needed" for the "Documentation" option, so we can also update our wallet docs with this information.